File: Conceptos Basicos/ejercicio04.py
numero = int(input("Ingresa un numero:"))


for i in range (0,numero):
    print(i)
    if i%3==0:
        print(f"{i} Es multiplo de 3 -> Fizz")
    elif i%5==0:
        print(f"{i} Es multiplo de 5 -> Buzz")
    else:
        print("Otra opcion")

File: Conceptos Basicos/ejercicio04.java
import java.util.Scanner;

public class ejercicio04 {
    public static void main(String[] arg){
        Scanner scan = new Scanner (System.in);
        
        System.out.println("Ingresa unu numero:");
        int numero= scan.nextInt();

        for (int i = 0; i<= numero; i++){
            if (i%3 ==0){
                 System.out.println(i+" Es multiplo de 3  -> Fizz");
            } else if (i%5==0){
                  System.out.println(i+" Es multiplo de 5  -> Buzz");
            }
            else{
                 System.out.println(i+" Otro caso");
            }
            
        }
    }
}
